start_validator.sh

```diff
@@ -43,14 +44,23 @@ pm2 start neurons/validator.py --name $VALIDATOR_PROCESS_NAME -- \
   --wallet.name $WALLET_NAME \
   --wallet.hotkey $WALLET_HOTKEY \
   --axon.port $VALIDATOR_AXON_PORT \
-  --proxy.port $VALIDATOR_PROXY_PORT \
-  --neuron.device $DEVICE
+  --proxy.port $VALIDATOR_PROXY_PORT
+
+# REAL DATA CACHE UPDATER PROCESS
+if pm2 list | grep -q "$CACHE_UPDATE_PROCESS_NAME"; then
+  echo "Process '$CACHE_UPDATE_PROCESS_NAME' is already running. Deleting it..."
+  pm2 delete $CACHE_UPDATE_PROCESS_NAME
+fi
+
+echo "Starting real data cache updater process"
+pm2 start bitmind/validator/scripts/run_cache_updater.py --name $CACHE_UPDATE_PROCESS_NAME
 
 # SYNTHETIC DATA GENERATOR PROCESS
 if pm2 list | grep -q "$DATA_GEN_PROCESS_NAME"; then
   echo "Process '$DATA_GEN_PROCESS_NAME' is already running. Deleting it..."
   pm2 delete $DATA_GEN_PROCESS_NAME
 fi
 
-echo "Starting SyntheticDataGenerator process"
-pm2 start bitmind/validator/run_data_generator.py --name $DATA_GEN_PROCESS_NAME
+echo "Starting synthetic data generation process"
+pm2 start bitmind/validator/scripts/run_data_generator.py --name $DATA_GEN_PROCESS_NAME -- \
+  --device $DEVICE
```

Comment:
device is now a synthetic data generator arg, since this is the process that uses gpu